src/SFA.DAS.EmployerApprenticeshipsService.Application/Mappings/MembershipMappings.cs

```diff
@@ -1,14 +0,0 @@
-﻿using AutoMapper;
-using SFA.DAS.Authorization;
-using SFA.DAS.EAS.Domain.Models.AccountTeam;
-
-namespace SFA.DAS.EAS.Application.Mappings
-{
-    public class MembershipMappings : Profile
-    {
-        public MembershipMappings()
-        {
-            CreateMap<Membership, MembershipContext>();
-        }
-    }
-}
```

Comment:
I am worried that this file is needed in the Jobs project

Reply:
Which jobs project @tom-gough ? `SFA.DAS.EmployerAccounts.Jobs` should be using the mappings from `SFA.DAS.EmployerAccounts/Mappings` should it not?

Reply:
Ah sorry, didn't see this was EAS mappings. Why just remove one mapping though?

Reply:
@tom-gough I don't know the definite answer to your last question, but as @MichaelYoung1981 was removing the _old_ `SFA.DAS.Auth..` stuff including the code that was checking membership and replacing with _new_ (nuget) `SFA.DAS.Auth` my guess is he's just gone through removing redundant refs to membership; mapping being one of them (as opposed to specifically tidying up mappings specifically) if that makes sense.

Reply:
fair enough, I guess that will get tidied up when we nuke any unneeded EAS stuff